SpinCore_pp/field_control.py

```diff
@@ -0,0 +1,23 @@
+from Instruments.XEPR_eth import xepr
+import pyspecdata as psd
+
+logger = psd.init_logging(level="info")
+
+def set_field(config_dict):
+    """Ensures the field is appropriate before setting the desired field."""
```

Comment:
The key point is that the field is set based on the carrier frequency and the estimated gamma